seumich/urls.py

```diff
@@ -30,10 +30,6 @@
         r'^cohorts/$',
         views.CohortsListView.as_view(),
         name='cohorts_list'),
-    url(
```

Comment:
A few lines down below, there is another classes url pattern. That can also be removed:
 
url(
        r'^classes/(?P<class_site_id>\d+)/$',
        views.ClassSiteView.as_view(),
        name='class_site'),

Reply:
That ClassSiteView is a separate view that shows the students enrolled in a specific course and isn't causing a problem here. I don't know if this URL is used either but I didn't remove it as there was no performance or other issue with it. 

I think @mfldavidson and I were talking about this view and she said this one might be useful but for some reason it's not using the Canvas ID like the URL

`    url(r'^students/(?P<student>\w+)/class_sites/(?P<classcode>[\w-]+)/$',
`

Is using, instead it uses the internal code which nobody knows. If it used the Canvas class code it could be a useful URL.

Reply:
Got it!  Will leave this in then.